modules/apps/questions/questions-web/src/main/resources/content/Language.properties

```diff
@@ -53,6 +53,7 @@ topic-name=Topic Name
 unmark-as-answer=Unmark As Answer
 update-your-answer=Update Your Answer
 update-your-question=Update Your Question
+use-the-topic-names-in-url=Use the topic names in URL
```

Comment:
I doubt about this... Reminds me about the sentence or title discussion. This should not be a title like "Use Topic Names in URL"?

Reply:
Oh! Yeah, you are right. My bad, sorry :( 
I Will change it.
Thanks!